pom.xml

```diff
@@ -89,7 +89,7 @@
       <dependency>
         <groupId>org.robolectric</groupId>
         <artifactId>android-all</artifactId>
-        <version>5.0.0_r2-robolectric-0</version>
+        <version>5.0.0_r2-robolectric-1</version>
```

Comment:
If you're going to modify the android-all jar, I think it would be better to have the layoutlib merging code in `build-android.sh` and pull the layoutlib from AOSP. I'm worried that someone will build android-all in the future and forget that there is another step to get a complete jar.